src/main/java/com/google/sps/data/CalendarClientData.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.*;
+
+/** Class containing the user's free hours. */
+public final class CalendarClientData {
+
+  private final int startDay;
+  private final int[] workHours;
+  private final int[] personalHours;
```

Comment:
nit: Communicating a [review comment](https://github.com/googleinterns/step7-2020/pull/63#discussion_r455246251) I received from Gui here, I think `List<int>` would be preferable even if that includes conversion.

Reply:
Changed to using `List`. `Resolved`